<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AssemblyFlowWidget</class>
 <widget class="QWidget" name="AssemblyFlowWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>989</width>
    <height>707</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="toolReadXml">
       <property name="text">
        <string>读取配置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolSaveXml">
       <property name="text">
        <string>保存配置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolNewStep">
       <property name="text">
        <string>新建步骤</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolCopyStep">
       <property name="text">
        <string>复制步骤</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolSaveStep">
       <property name="text">
        <string>保存步骤</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolDelStep">
       <property name="text">
        <string>删除步骤</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTreeWidget" name="treeWidget">
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QtTreePropertyBrowser" name="propertyBrowser" native="true"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtTreePropertyBrowser</class>
   <extends>QWidget</extends>
   <header location="global">qttreepropertybrowser.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>toolReadXml</tabstop>
  <tabstop>toolSaveXml</tabstop>
  <tabstop>toolNewStep</tabstop>
  <tabstop>toolCopyStep</tabstop>
  <tabstop>toolSaveStep</tabstop>
  <tabstop>toolDelStep</tabstop>
  <tabstop>treeWidget</tabstop>
  <tabstop>tableWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
